core/server/common/src/main/java/alluxio/master/journal/raft/LocalFirstRaftClient.java

```diff
@@ -70,7 +74,7 @@ public LocalFirstRaftClient(RaftServer server, Supplier<RaftClient> clientSuppli
    */
   public CompletableFuture<RaftClientReply> sendAsync(Message message,
       TimeDuration timeout) throws IOException {
-    if (mClient == null) {
+    if (!mEnableRemoteClient || mClient == null) {
```

Comment:
WRITE_REMOTE_ENABLED == false && WRITE_LOCAL_FIRST_ENBLAED == false send local request
WRITE_REMOTE_ENABLED == true &&  WRITE_LOCAL_FIRST_ENBLAED == false send remote request
WRITE_REMOTE_ENABLED == false &&  WRITE_LOCAL_FIRST_ENBLAED == true send local request
WRITE_REMOTE_ENABLED == true &&  WRITE_LOCAL_FIRST_ENBLAED == true send local request

so only WRITE_REMOTE_ENABLED == true &&  WRITE_LOCAL_FIRST_ENBLAED == false send remote request
two properties are somehow a little difficult to understand what's desire, can we consolidate them into one property and add a description

Reply:
`WRITE_REMOTE_ENABLED` isn't a property intent for user configuration in normal cases. It is only in case that users see issues with local only writes then they can re-enable the old behavior. Once the new behavior is validated over time, we should remove the entire remote write logic and issue only local writes.

Reply:
Oh got it, then could you update the description for write_remote_enabled to include more information (like when it should be used), thanks

Reply:
Added some more description.